core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -332,17 +359,12 @@ public PacketSignal handle(ResourcePackChunkRequestPacket packet) {
 
         session.sendUpstreamPacket(data);
 
-        // Check if it is the last chunk and send next pack in queue when available.
-        if (remainingSize <= GeyserResourcePack.CHUNK_SIZE && !packsToSent.isEmpty()) {
-            sendPackDataInfo(packsToSent.pop());
-        }
-
```

Comment:
We send these packets to for (String id : packet.getPackIds()) { on line 232, can this cause spaces?  I tested everything fine, it was downloaded from my phone and computer, it was removed so that they could not overfill the list with garbage and errors